seven-days-to-dieupdates.json

```diff
@@ -71,5 +71,87 @@
     "UpdateSourceConditionSetting": "UMOD",
     "UpdateSourceConditionValue": "true",
     "DeleteAfterExtract": true
+  },
+  {
+    "UpdateStageName": "Settings File Download",
+    "UpdateSourcePlatform": "All",
+    "UpdateSource": "FetchURL",
+    "UpdateSourceData": "https://github.com/gkuba/AMPTemplates/raw/main/seven-days-settings-undead-legacy-stable.xml",
+    "UpdateSourceTarget": "./294420",
+    "OverwriteExistingFiles": false,
+    "UpdateSourceConditionSetting": "ConfigFile",
+    "UpdateSourceConditionValue": "seven-days-settings-undead-legacy-stable"
+  },
+  {
+    "UpdateStageName": "Settings File Download",
+    "UpdateSourcePlatform": "All",
+    "UpdateSource": "FetchURL",
+    "UpdateSourceData": "https://github.com/gkuba/AMPTemplates/raw/main/seven-days-settings-undead-legacy-experimental.xml",
+    "UpdateSourceTarget": "./294420",
+    "OverwriteExistingFiles": false,
+    "UpdateSourceConditionSetting": "ConfigFile",
+    "UpdateSourceConditionValue": "seven-days-settings-undead-legacy-experimental"
+  },
+  {
+    "UpdateStageName": "Settings File Download",
+    "UpdateSourcePlatform": "All",
+    "UpdateSource": "FetchURL",
+    "UpdateSourceData": "https://ul.subquake.com/dl?v=stable",
+    "UpdateSourceArgs":"UndeadLegacyStable.zip",
+    "UpdateSourceTarget": "{{$FullBaseDir}}",
+    "UnzipUpdateSource": true,
+    "OverwriteExistingFiles": true,
+    "UpdateSourceConditionSetting": "UndeadLegacyStable",
+    "UpdateSourceConditionValue": "true",
+    "DeleteAfterExtract": true
+  },
+  {
+    "UpdateStageName": "Settings File Download",
```

Comment:
The name doesn't match the task it's performing.

Reply:
Fixed changed to "Download Undead Legacy Stable"